### ISSUE TITLE
the role of cam_up_similarity

### ISSUE THREAD

**Issue by yuyouxixi** [mbortolon97/6dgs#7]
Hi, thank you for your wonderful work! I notice that in the training process, the loss function is composed of the attention loss and the cam_up_similarity. Since the cam_up_similarity was not discussed in the paper, may I ask what is the role of  cam_up_similarity?

**Comment by mbortolon97** [comment]
The cam_up_similarity is a simple loss that predicts the up direction of the model from the image. This is necessary to remove the last degree of freedom when predicting the rotation matrix. While you can predict it directly from the attention map, we notice that especially in some scenes this solution tends to be more stable at training time, but still with results extremely close to the ones published.

**Comment by yuyouxixi** [comment]
Thank you for your reply. I am still confused about how to predict rotation matrix directly from the attention map. Can you help me? Thanks!
